packages/builder/src/components/integration/SvelteEditor.svelte

```diff
@@ -0,0 +1,188 @@
+<script>
+  import CodeMirror from "./codemirror"
```

Comment:
NB: this is copied from the svelte REPL. I don't think we need a lot of this code